dev/src/lobster/typecheck.h

```diff
@@ -219,6 +219,22 @@ struct TypeChecker {
         }
     }
 
+    bool ConvertsToStatically(TypeRef type, TypeRef bound) {
+        if (bound == type || type->t == V_ANY)
+            return true;
+        if (bound->t != V_CLASS || type->t != V_CLASS)
+            return false;
+        if (st.SuperDistance(type->udt, bound->udt) > -1)
```

Comment:
ConvertsTo checks if it's possible to happen at runtime and checks if it's a sane downcast, while ConvertsToStatically asserts that it will be so for sure at compile time and thus looks checks if it's an upcast

Reply:
as per #99 I'll remove this